ci/jenkins/test.sh

```diff
@@ -141,6 +143,8 @@ function clean_antrea {
     for antrea_yml in ${WORKDIR}/*.yml; do
         kubectl delete -f $antrea_yml --ignore-not-found=true || true
     done
+    docker images | grep 'antrea' | awk '{print $3}' | xargs -r docker rmi || true
+    docker images | grep '<none>' | awk '{print $3}' | xargs -r docker rmi || true
```

Comment:
doesn't `docker image prune -a` work?

Reply:
We don't want to clean all images. Some images like busybox can be reused in CI test.